src/System.Linq/tests/ZipTests.cs

```diff
@@ -0,0 +1,393 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Copied to new location, and then edited in place, before deleting the original after that was finished, so I could check I hadn't missed on of the cases in the original.